### ISSUE TITLE
BUG: MongoDB SSH Tunnel Doesn't work

### ISSUE THREAD

**Issue by upbeta01** [dbgate/dbgate#106]
**Describe the bug**
I wish I could tell you more about this error I'm getting trying to use DBGate with SSH Tunnel enabled.
The general error says: **Connect failed:  All configured authentication methods failed**

The full error says:
```
Error: All configured authentication methods failed
    at e.exports (/opt/dbgate/resources/app.asar/packages/api/dist/bundle.js:15:9303)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async process.<anonymous> (/opt/dbgate/resources/app.asar/packages/api/dist/bundle.js:191:413569)

Error JSON: {}

Connection: {
  "server": "***",
  "engine": "mongo@dbgate-plugin-mongo",
  "sshMode": "keyFile",
  "sshPort": "***",
  "sshKeyfile": "***",
  "user": "***",
  "password": "***",
  "passwordMode": "saveEncrypted",
  "defaultDatabase": "***",
  "useSshTunnel": true,
  "sshHost": "***",
  "sshLogin": "***"
}

Platform: linux
```

**To Reproduce**
- To reproduce, setup a MongoDB in private network with SSH tunnel enabled from a host on running in a public network..
- PS: This setup is working using robo3t

**Expected behavior**
It should enable authentication.

**Version Information (please complete the following information):**
 - OS: Linux (MXLinux,  Debian10)
 - App Version: Version: 4.1.10 | Build date: 2021-04-27
 - Install source: .deb package
 - Type: Application
 - Database engine: MongoDB

**Comment by janproch** [comment]
Thank you for the error report. This issue was related to SSH auth with keyfile, is fixed in newest beta version.

**Comment by upbeta01** [comment]
Thanks @janproch . Good to know and I look forward for the release. More power to this project.

**Comment by janproch** [comment]
fixed in version 4.1.11. In case of problems please let me know. Closing for now
